modules/ROOT/examples/live-demos/autocorrect/index.html

```diff
@@ -0,0 +1,10 @@
+<div id="tiny-ui">
+  <textarea id="autocorrect" style="width: 100%; height: 500px;">
+    <h2>Welcome to Spelling Autocorrect.</h2>
+    <p>Please try this demo of the Spelling Autocorrect premium plugin.</p>
+    <ol>
+    <li>Deliberately mis-type a word. For example, type <em>teh</em> for <em>the</em>.</li>
```

Comment:
I think that it might actually be `mistype`, without the hyphen. Let's ask the expert: @brian-forte

Reply:
I think `type <em>teh</em> instead` might be clearer.

Reply:
I went with `mistype` for now